### DISCUSSION TITLE
求教如何跳过 xmake r 对 target 的设置，直接指定运行参数

### DISCUSSION THREAD

**Discussion by MasterBalll** [xmake-io/xmake#3510] [Q&A]
如题。

**Comment by star-hengxing** [comment]
这个[set_runargs](https://xmake.io/#/zh-cn/manual/project_target?id=targetset_runargs)？

**Reply by MasterBalll** [reply]
`set_runargs()` 应该不大管用。
因为有批量运行需求，要在命令行设置不同的运行参数。

**Reply by waruqi** [reply]
target name 是省不掉，你必须指定，除非你自己搞个 custom task 来处理

**Reply by MasterBalll** [reply]
好吧，我再想想办法。
